src/sessions.ts

```diff
@@ -169,8 +169,10 @@ export class ClientSession extends TypedEventEmitter<ClientSessionEvents> {
     this[kServerSession] = this.explicit ? this.sessionPool.acquire() : null;
     this[kTxnNumberIncrement] = 0;
 
+    const defaultCausalConsistencyValue = this.explicit && options.snapshot !== true;
     this.supports = {
```

Comment:
I caught myself up on the spec work and all, just double checking here there is no concern that the default is changing because the goal is to address a bug with the noted read concern settings? (I see the bullet in the description but cannot be to cautious)

Reply:
the default is changing for the implicit sessions only; the default for the explicit sessions is the same as it has been (true for all except snapshot-enabled), and the unit tests are there to express the expected behavior